runnable.go

```diff
@@ -0,0 +1,60 @@
+package compute
+
+import (
+	"fmt"
+	"sync"
+)
+
+type Runnable struct {
+	environment  string
+	customerID   string
+	namespace    string
+	functionName string
+	version      string
+
+	token string
+
+	lock sync.Mutex
```

Comment:
I don't think we need a lock on this, I highly doubt any highly-concurrent code is going to be doing anything with it :)